README.rst

```diff
@@ -178,6 +178,14 @@ Contributing to DALI
 
 Contributions to DALI are more than welcome. To make the pull request process smooth, please follow these `guidelines <CONTRIBUTING.md>`_.
 
+Reporting problem, asking questions
```

Comment:
Typo: I suppose, there should be "problems"

Reply:
Done